superdesk/celery_app.py

```diff
@@ -168,18 +168,20 @@ def update_key(key, flag=False, db=None):
 
 def _update_subtask_progress(task_id, current=None, total=None, done=None):
     redis_db = redis.from_url(celery.conf['CELERY_RESULT_BACKEND'])
```

Comment:
Yeah but it could b=maybe throw some other exception if it can't open a connection properly. Can check if it's set when trying to close it in finally.